streamer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.10-slim
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+COPY ./streamer/requirements.txt /app/
+COPY ./streamer/iceberg_streamer.py /app/
+
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
I recommend putting this before the iceberg_streamer copy. Otherwise whenever you change the python file you'll have to reinstall dependencies.